### PR TITLE
Css selector

### PR DESCRIPTION
**Pull request by JesusPaz**

### CODE REVIEW

**Comment by JesusPaz** [comment]
Implement the POM pattern

File: src/page/shipping-step.page.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    this.toCheckoutBtn = $('#form > p > button > span');
+  }
+
+  public async agreeTermsCheckClick(): Promise<void> {
+    await this.agreeTermsCheck.click();
+  }
+  public async toCheckoutBtnClick(): Promise<void> {
+    await this.toCheckoutBtn.click();
+  }
+
```
Comment (holgiosalos) [inline src/page/shipping-step.page.ts:18]: delete this blank line

**Review by yesid-lopez — APPROVED** [review]

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -1,41 +1,53 @@
...
+  const menuContentPage: MenuContentPage = new MenuContentPage();
+  const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
+  const productListPage: ProductListPage = new ProductListPage();
+  const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
+  const singInStepPage: SingInStepPage = new SingInStepPage();
+  const addressStepPage: AddressStepPage = new AddressStepPage();
+  const shippingStepPage: ShippingStepPage = new ShippingStepPage();
+  const paymentStepPage: PaymentStepPage = new PaymentStepPage();
+  const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
   beforeEach(() => {
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts:16]: add a blank line to separate constructors from beforeEach

File: src/page/sign-in-step.page.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  constructor () {
+    this.emailKey = $('#email');
+    this.passwordKey = $('#passwd');
+    this.submitBtn = $('#SubmitLogin > span');
+  }
+
+  public async sendEmailKeys(): Promise<void> {
+    await this.emailKey.sendKeys('aperdomobo@gmail.com');
+  }
+  public async sendPsswordKeys(): Promise<void> {
```
Comment (holgiosalos) [inline src/page/sign-in-step.page.ts:17]: separate all methods with a blank line between them

**Review by holgiosalos — CHANGES_REQUESTED** [review]

**Pull request merged**